origin-dapp/src/components/listing-create.js

```diff
@@ -165,7 +170,11 @@ class ListingCreate extends Component {
       try {
         // Pass false as second param so category doesn't get translated
         // because the form only understands the category ID, not the translated phrase
-        const listing = await getListing(this.props.listingId, false)
+        const listing = await getListing(this.props.listingId, { translate: false, loadOffers: true })
+        if (listing.isMultiUnit) {
+          listing.boostLimit = listing.totalBoostValue
+          delete listing.totalBoostValue
```

Comment:
why is this `delete` needed?

Reply:
Thanks for pointing to this. The logic was that we don't allow editing commission / commission per unit, and those fields should for that reason be removed. But now that I think about it that logic is flawed. We should load both commissions and leave them unchanged when submitting an edited listing.

removing this if condition